postprocessing/comstockpostproc/comstock.py

```diff
@@ -1992,21 +2050,29 @@ def create_geospatially_resolved_aggregations(self, geographic_col_name, pretty_
             logger.warn("ulimit -n 200000")
             logger.warn("ulimit -u 2048")
             logger.info("Attempting pottentially OSERROR triggering write:")
-            to_write.collect().write_parquet(file_path, use_pyarrow=True, pyarrow_options={"partition_cols": [pretty_geo_col_name], 'max_partitions': 3143})
+            to_write.collect().write_parquet(file_path, use_pyarrow=True, pyarrow_options={
+                "partition_cols": [pretty_geo_col_name], 'max_partitions': 3143
+            })
 
         # Export dictionaries corresponding to the exported columns
         self.export_data_and_enumeration_dictionary()
 
+        # Return the geospatially aggregated dataframe for use by plotting, etc.
+        return spatial_aggregation
 
     def add_weights_aportioned_by_stock_estimate(self, apportionment: Apportion, keep_n_per_apportionment_group=False):
+        # This function doesn't support already CBECS-weighted self.data - error out
+        if self.CBECS_WEIGHTS_APPLIED:
```

Comment:
GREAT!

Reply:
Actually there is a even shorter way:

assert self.CBECS_WEIGHTS_APPLIED == True , "Unable to apply apportionment weighting after CBECS weighting - reverse order."